scripts/build_documentation.py

```diff
@@ -0,0 +1,67 @@
+"""Build Fairlearn documentation
+
+The static landing page is no longer under the control of Sphinx,
+since we only want one copy of it as we enable multiple documentation
+versions.
+
+This makes the documentation build a three-stage process:
+1. Copy the static pages into the output directory
+2. Do the sphinx build
+3. Make a duplicate copy of a single PNG in the output directory (a logo)
+
+This ordering is in part because shutil.copytree() only acquired the
+dirs_exist_ok argument in Python 3.8
+"""
+
+import argparse
+import logging
+import os
+import shutil
+import subprocess
+import sys
+
+from _utils import _ensure_cwd_is_fairlearn_root_dir, _LogWrapper
+
+_logger = logging.getLogger(__file__)
+logging.basicConfig(level=logging.INFO)
+
+landing_page_directory = "static_landing_page"
+extra_png_src_path = os.path.join("_static", "images", "fairlearn_full_color.png")
+
+
+def _build_argument_parser():
+    desc = "Build widget for fairlearn dashboard"
+
+    parser = argparse.ArgumentParser(description=desc)
+
+    parser.add_argument("--documentation-path",
+                        help="The path to the documentation sources (conf.py directory)",
+                        required=True)
+
+    parser.add_argument("--output-path",
+                        help="The directory for the output files (will be created)",
+                        required=True)
+    return parser
+
+
+def main(argv):
+    _ensure_cwd_is_fairlearn_root_dir()
+    parser = _build_argument_parser()
+    args = parser.parse_args(argv)
+
+    with _LogWrapper("Copy static files"):
```

Comment:
super-nit: `_LogWrapper` prints this as "start <string>" so this would be "start Copy static files", when "start copying static files" is probably easier to read. Yeah I know... that's why super-nit :-)